src/main/resources/referenceImageIDTemplateWithManagedDisk.json

```diff
@@ -88,6 +88,8 @@
                     },
                     "osDisk": {
                         "createOption": "FromImage",
+                        "caching": "[if(bool(variables('ephemeralOSDisk')), 'ReadOnly', json('null'))]",
+                        "diffDiskSettings": "[if(bool(variables('ephemeralOSDisk')), json('{\"option\": \"Local\"}'), json('null'))]",
```

Comment:
Only this json file has `diffDiskSettings` property?

Reply:
enabled this wherever ephemeral disk is in use per https://docs.microsoft.com/en-us/azure/templates/microsoft.compute/2019-07-01/virtualmachines#DiffDiskSettings